src/d3a/models/market/two_sided.py

```diff
@@ -373,3 +387,14 @@ def replace_recommendations_with_residuals(recommendation: Dict):
             recommendations = [replace_recommendations_with_residuals(recommendation)
                                for recommendation in recommendations]
         return recommendations
+
+
+class SettlementMarket(TwoSidedMarket):
+
+    @property
+    def _class_name(self):
+        return "Settlement"
+
+    @property
+    def _debug_log_market_type_identifier(self):
+        return "[SETTLEMENT]"
```

Comment:
How about creating a module `src/d3a/models/market/settlement_market.py` and moving it there. Although now now `SettlementMarket ` is only a wrapper of `TwoSidedMarket `

Reply:
Works for me, will do so